web/modules/custom/joinup_notification/joinup_notification.behat.inc

```diff
@@ -119,7 +119,7 @@ class JoinupNotificationSubContext extends DrupalSubContextBase implements Drupa
       // In non user emails, html p tags are used for new lines and these are
       // stripped instead of replaced. Thus, a regex will be used to properly
       // identify the signature regardless of spaces.
-      if (!preg_match('/Kind regards,\s{0,2}The Joinup Support Team/', $mail_body)) {
+      if ($mail['key'] !== 'contact_form_submission' && !preg_match('/Kind regards,\s{0,2}The Joinup Support Team/', $mail_body)) {
```

Comment:
Here we are hardcoding that contact form submissions do not have the "Kind regards" signature. This should be moved to the Behat test scenario, so that it is clearly functionally described that this notification does not have a signature.

So in the Behat test we can have something like this:

```
Then the following email should have been sent:
  | template  | Contact form submission          |
  | signature | -no signature-                   |
```

And over here something like:

```
$signature = !empty($mail['signature']) ? $mail['signature'] : 'Kind regards, \s{0,2}The Joinup Support Team';
if ($signature !== '-no signature-' && !preg_match(...) {}
```

Also make sure to document this option in the method docblock.